eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -30,17 +31,27 @@ class ContentFieldValidationException extends APIContentFieldValidationException
      */
     protected $errors;
 
+    /** @var string|null */
+    protected $target;
+
     /**
      * Generates: Content fields did not validate.
      *
      * Also sets the given $fieldErrors to the internal property, retrievable by getFieldErrors()
      *
      * @param array<array-key, array<string, \eZ\Publish\Core\FieldType\ValidationError>> $errors
      */
-    public function __construct(array $errors)
+    public function __construct(array $errors, ?string $target = null)
```

Comment:
Thinking about this further, maybe here should be rather `?ContentInfo`? It's a value and this exception is related to Content, so I don't see mixed layers issue here. The reason here is that I'd rather expect `[<content_id>] <content_name>` pattern in error message, as the content name by itself might be to ambiguous.

Reply:
Tbh `content_id` gives not much of a value. For example, when we are copying a subtree or content it will just be a meaningless number as it would be just an ID of a newly created Content ID. But if you still believe that we should place it there I'm fine with it.